feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/radio_ubus.c

```diff
@@ -100,23 +102,30 @@ int ubus_set_channel_switch(const char *if_name, uint32_t frequency,
 
 	if (ubus_lookup_id(ubus, path, &id))
 		return -1;
-	blob_buf_init(&b, 0);
-
-	if (channel_bandwidth == 20 || channel_bandwidth == 40) {
-		blobmsg_add_u8(&b, "ht", 1);
-	} else if (channel_bandwidth == 80) {
-		blobmsg_add_u8(&b, "vht", 1);
-	}
-
-	if (channel_bandwidth == 40 || channel_bandwidth == 80) {
-		blobmsg_add_u32(&b, "center_freq1", frequency+30);
+	blob_buf_init(&ub, 0);
+
+	if (!strncmp(hw_mode, "11n", strlen("11n"))) {
```

Comment:
Do we need only 1 of these flags set or should HE include VHT and HT?

Test would be that an 11n only client works with the 11ac or 11ax mode AP.

Reply:
Yes, The flags are to be enabled including the lower modes.